index.js

```diff
@@ -1,25 +1,38 @@
-'use strict';
+const bunyan = require('bunyan');
+const uuid = require('uuid');
+const util = require('util');
+const onFinished = require('on-finished');
 
-var bunyan = require('bunyan');
-var uuid = require('uuid');
-var util = require('util');
-var onFinished = require('on-finished');
+const updateFields = (ctx, func, data, err) => {
+  if (!func) return data;
+
+  try {
+    if (err) {
+      return func.call(ctx, data, err) || data;
+    } else {
+      return func.call(ctx, data) || data;
+    }
+  } catch (e) {
+    ctx.log.error(e);
+    return data;
+  }
+};
 
 /*
  * If logger is a bunyan logger instance, return it;
  * otherwise, create a new logger with some reasonable defaults.
  */
-function createOrUseLogger(logger) {
+const createOrUseLogger = logger => {
```

Comment:
I sort of like the idea of having () around the args .. so = (logger) => {}  foo?  Many linters sort of demand this ..

Reply:
true that but we don't have any modern lining rules yet, so we can agree on such rule, will update the PR, the point is not even having parantheses or not, but have consistent codebase

![0_dwiyyjzi5mu13l_m](https://user-images.githubusercontent.com/5700359/46399691-c2f7aa00-c6f8-11e8-8efc-9db09fb8ef83.png)

Reply:
LOL -- too true.  And I am looking at your LINT rules too :)